opcua.cpp

```diff
@@ -263,6 +295,28 @@ OPCUA::setSecPolicy(const std::string& secPolicy)
 	}
 }
 
+/**
+ * Create an S2OPC OPCUA Toolkit trace file name if requested
+ *
+ * @param value    If true, create an S2OPC OPCUA Toolkit trace file
+ */
+void
+OPCUA::setTraceFile(const std::string& traceFile)
+{
+	if (traceFile == "True" || traceFile == "true" || traceFile == "TRUE")
+	{
+		string traceFilePath = getDataDir() + string("/logs/");
```

Comment:
It would be better to put this to the syslog such that it is then available via the UI as opposed to a file that people will need to access the box for.